drivers/spi/spis_nrf5.c

```diff
@@ -0,0 +1,467 @@
+/* spis_nrf5.c - SPI slave driver for Nordic nRF5x SoCs */
+/*
+ * Copyright (c) 2016, 2017 Linaro Limited.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#define SYS_LOG_LEVEL CONFIG_SYS_LOG_SPI_LEVEL
+
+#include <device.h>
+#include <errno.h>
+
+#include <logging/sys_log.h>
+#include <sys_io.h>
+#include <board.h>
+#include <init.h>
+#include <gpio.h>
+#include <spi.h>
+#include <nrf.h>
+#include <toolchain.h>
+
+typedef void (*spis_nrf5_config_t)(void);
+
+struct spis_nrf5_config {
+	NRF_SPIS_Type *regs;			/* Registers */
```

Comment:
use spaces in between instead of tabs to get comment alignment consistent all over this file.